README.md

```diff
@@ -1,2 +1,16 @@
 # ChimbukoVisualizationII
+
 Scalable visualization module for Chimbuko
+
+## Overview 
+
+![Overview](./web/static/images/overview.png)
+
+### In-situ Visualization Components
+* `Top & Bottom Dynamic MPI Ranks`: visualizes the most and the least fluctuating MPI ranks simultaneously based on several statistical values, such as min, max, mean, std, skewness, kurtosis, and cumulative delta.
+* `Selected Rank History`: visualizes temporal anomaly distribution of the selected MPI rank, which can be navigated from the beginning to the latest frame. 
+
+### Online Analysis Visualization Components
```

Comment:
This is still experimental. Codes related to online analysis view need to be reviewed later.